app/presenters/source_presenter.rb

```diff
@@ -22,6 +22,7 @@ def as_json(opts = {})
       status: source.status,
       is_review: source.is_review?,
       clinical_trials: source.clinical_trials.map { |t| ClinicalTrialPresenter.new(t) },
+      evidence_item_count: source.evidence_items.size,
```

Comment:
This adds a `select count(*) from evidence items....` to each source presenter we create, so anywhere we have a list of them we'll get an `n+1` query issue. That may be rare enough we don't care, but its something to be aware of.

Reply:
Since this is only needed for the advanced search results, we could make a new presenter that includes this, leaving the original SourcePresenter unchanged.